packages/ckeditor5-image/src/imageupload/imageuploadediting.ts

```diff
@@ -572,6 +572,8 @@ export class ImageUploadEditing extends Plugin {
 }
 
 /**
+ * TODO move this to the clipboard package.
```

Comment:
This deserves an issue to track.

Reply:
https://github.com/ckeditor/ckeditor5/issues/18785 issue created